staging/src/k8s.io/client-go/readme_published.md

```diff
@@ -0,0 +1,10 @@
+### Layout
```

Comment:
rename this file to just be README.md

Add a section explaining what the kubernetes go client is, what it includes, how to use it, that you should be able to go get it, and where to report bugs.